src/insns/cmove_32bit.adoc

```diff
@@ -1,24 +1,34 @@
 <<<
 
-[#CMOVE,reftext="CMOVE"]
-==== CMOVE
+[#CMV,reftext="CMV"]
+==== CMV
 
 include::new_encoding_note.adoc[]
 
+ifdef::cheri_v9_annotations[]
```

Comment:
Hmm actually this file should be deleted?

Reply:
will do